mattermost-plugin/plugin.json

```diff
@@ -6,8 +6,8 @@
     "support_url": "https://github.com/mattermost/focalboard/issues",
     "release_notes_url": "https://github.com/mattermost/focalboard/releases",
     "icon_path": "assets/starter-template-icon.svg",
-    "version": "7.8.0",
-    "min_server_version": "7.2.0",
+    "version": "7.8.1",
+    "min_server_version": "7.8.0",
```

Comment:
I'm curious why min server version was 7.2 but now 7.8 for a minor release?

Reply:
Because we have this issue (yet to be cp'ed) that will be going in. 
https://github.com/mattermost/focalboard/pull/4542

It is dependent on this server PR. 
https://github.com/mattermost/mattermost-server/pull/22226